### PR TITLE
create org_members table & relation

### PR DESCRIPTION
**Pull request by levity**
create the `org_members` table and the backend for direct relationships between orgs & profiles

reviewers: check out [the test file](https://github.com/coordinape/coordinape/pull/1944/files#diff-39a26b45001fe716b4258bad2ecdd55f349a209ceae6fd36f77f172e8677ee88) for a pretty readable summary of what this enables

### CODE REVIEW

**Comment by levity** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1943** <a href="https://app.graphite.dev/github/pr/coordinape/coordinape/1943" target="_blank">Graphite</a> 
    * **PR #1944** <a href="https://app.graphite.dev/github/pr/coordinape/coordinape/1944" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/coordinape/coordinape/1944?utm_source=stack-comment).

File: hasura/migrations/default/1676345211557_org_members/up.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE public.org_members (
+    id BIGSERIAL PRIMARY KEY,
```
Comment (CryptoGraffe) [inline hasura/migrations/default/1676345211557_org_members/up.sql:2]: This column is probably unnecessary, but its not a big deal

Comment (levity) [reply hasura/migrations/default/1676345211557_org_members/up.sql:2]: it's a bit more future-proof -- we'll need it if we ever want to have something have a foreign key to this table

**Comment by CryptoGraffe** [comment]
Compound foreign key no? Just philosophical like I said don’t really care

**Comment by levity** [comment]
Simple foreign keys are more efficiently indexed

**Review by CryptoGraffe — APPROVED** [review]

File: hasura/migrations/default/1676345211557_org_members/up.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    profile_id bigint NOT NULL REFERENCES profiles(id) ON DELETE RESTRICT ON UPDATE RESTRICT,
+    org_id bigint NOT NULL REFERENCES organizations(id) ON DELETE RESTRICT ON UPDATE RESTRICT,
+    deleted_at timestamp without time zone,
+    created_at timestamp without time zone NOT NULL DEFAULT now(),
+    updated_at timestamp without time zone,
```
Comment (crabsinger) [inline hasura/migrations/default/1676345211557_org_members/up.sql:7]: @levity can you add a trigger to this field to update on row changes, as we do with all other tables.

**Pull request merged**